content/ember/v4/auto-location.md

```diff
@@ -7,29 +7,31 @@ since: '4.1.0'
 
 #### Background
 
-Several years ago only some browsers supported the (then) new History Location API.
-Others could only serialize the router location into an url hash `my/path#/ember/route`.
+Several years ago (in Ember v1 days) few browsers supported the History Location API,
+and other browsers could only serialize the router location into an URL hash `my/path#/ember/route`.
 
-To handle this dynamically, Ember built an AutoLocation that would feature-detect
-and use either 'hash' or 'history' as the underlying mechanism.
+To handle this dynamically, Ember had an AutoLocation class that would use feature-detection to 
+determine if the browser supported the History Location API, and use either 'hash' or 'history' 
+as the underlying mechanism.
 
-These days, virtually all browsers support the history API, so this is what 'auto'
-will resolve to in almost every case. Since 'auto' has served its purpose, it's being removed.
+These days, virtually all browsers support the history API, so 'auto' will always resolve to 
+'history' as the location mechanism. Since 'auto' has served its purpose, it's being removed.
 
 #### Required Changes
 
-Set `locationType` in `config/environment.js` to `'history'`. This is it,
-your app should work just like it used to.
-
-Unless you know for sure that you need to keep feature detection in place (few need that),
-or that you'd like to use the 'hash' location strategy (rarely used in browsers,
-but can be useful for mobile apps delivered via a webview).
+Set `locationType` in `config/environment.js` to `'history'`. This is it. 
```

Comment:
I suggest showing a before/after of the `environment.js` file, something like:

Before:

```
// config/environment.js
'use strict';

module.exports = function (environment) {
  let ENV = {
    modulePrefix: 'my-app',
    environment,
    rootURL: '/',
    locationType: 'auto',
    …
};
```

After:


```
// config/environment.js
'use strict';

module.exports = function (environment) {
  let ENV = {
    modulePrefix: 'my-app',
    environment,
    rootURL: '/',
    locationType: 'history',
    …
};
```

Reply:
Good idea, I've added something along those lines.